js/Application.js

```diff
@@ -112,12 +148,12 @@ define(
 				// handle select all
```

Comment:
Can we get rid off the model-specific stuff from the core application component? And place the logic next to the specific modules? E.g. create requirejs components, include the logic there and require those components somewhere in appropriate route files?